paien.yaml

```diff
@@ -293,7 +293,7 @@ packages:
       - hdf5@1.10.1+szip+mpi+hl+fortran+cxx
       - scotch@6.0.4+esmumps+mpi~metis
 # TODO: update scorep as described in issue SL-557
```

Comment:
Could you also delete this comment? I think your commit addresses the issue in question.